test/integration/TestClient.ts

```diff
@@ -276,4 +324,70 @@ export class TestClient {
       });
     });
   }
+
+  private async removeWindowsDB(dbPath): Promise<void> {
```

Comment:
You don't need the async here...

```suggestion
  private async removeWindowsDB(dbPath): Promise<void> {
```